### PR TITLE
feat: inventory-tags implemented in gravestone interface

### PR DESCRIPTION
**Pull request by arnaudperalta**
Closes https://github.com/runelite/runelite/issues/14086

Gravestone widget added in the overlay.

### CODE REVIEW

**Comment by Adam-** [comment]
The purpose of `showOnInventory` is for interfaces which either are the inventory, or replace the inventory when a modal is open in order to alter the inventory options (which is how banks, shops, etc work). To the player these are generally indistinguishable. 

The grave stone reclaim interface however is a full modal in the center of the viewport? Which isn't an inventory interface.

So instead it would be better to just `showOnInterfaces()` the gravestone interface.

**Comment by arnaudperalta** [comment]
I understand the problem with my MR. Can I solve it by adding this line : `showOnInterfaces(GRAVESTONE_GROUP_ID);` in the InventoryTagsOverlay constructor instead of my modification in the WidgetItemOverlay class ? It is working as well.

**Comment by Nightfirecat** [comment]
Yes, that is what he suggested.

File: runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsOverlay.java [hunk 1]
```diff
@@ -38,6 +38,8 @@
 import net.runelite.client.util.ColorUtil;
 import net.runelite.client.util.ImageUtil;
 
+import static net.runelite.api.widgets.WidgetID.GRAVESTONE_GROUP_ID;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsOverlay.java:41]: Move this import between the `javax.inject.Inject` and `net.runelite.api.widgets.WidgetItem` imports; we don't separate static and non-static imports.

Comment (arnaudperalta) [reply runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsOverlay.java]: Done !

**Pull request merged**